test/presubmit-tests.sh

```diff
@@ -35,8 +35,6 @@ function post_build_tests() {
     echo "*** Checking ${makefile}"
     make -n -C $(dirname ${makefile}) || { failed=1; echo "--- FAIL: ${makefile}"; }
   done
-  subheader "Checking config files"
-  make -C config/prow test || { failed=1; echo "--- FAIL"; }
```

Comment:
As inappropriate as it is for this to have been in the Makefile to begin with, wouldn't an easier solution be to just run `make -C config/prow-staging test` as well?

Reply:
Staging Prow does not have Testgrid config file. I don't want to add a special logic in the Makefile, as it's already complicated enough :(

Reply:
k